skops/_min_dependencies.py

```diff
@@ -13,7 +13,7 @@
 dependent_packages = {
     "scikit-learn": ("0.24", "install", None),
     "scikit-learn-intelex": ("2021.7.1", "docs", None),
-    "huggingface_hub": ("0.10.1", "install", None),
+    "huggingface_hub": ("0.18.0.rc0", "install", None),
```

Comment:
should this change before merge?

Reply:
Yes, the actual min version is 0.17. I can change it now or we can first ensure that all tests pass for 0.18 too -- which may take some time ;-)

Reply:
Okay, I'll let you know in a month or two.

Reply:
Is it really that bad? So it's not something we can maintain. Let's fix the version merge here, then we deal with the deprecation

Reply:
Yeah, the error now says that the daily quota has been exceeded. I don't remember it being this tight, not sure if it was reduced. We may be able to avoid it by using a HF employee token, but then we have the old issue of hiding it while still allowing it to be used in PRs, right?

Reply:
At the end of the day, it's not a free service, and it doesn't match the vibe of this library anymore I would say. We added it because everybody could simply use it, but with the new rate limits, I don't see why we would keep supporting it.

Reply:
I agree, the rate limit is pretty aggressive (still couldn't make all tests pass), so for practical purposes, it's not a free service anymore. I think we have a few options, I'll leave the decision up to you:

1. Still merge this PR, as I'm pretty sure it works (after adjusting the min version of course). Then create a new PR to deprecate the feature.
2. Don't merge this PR. Deprecate the feature.
3. Remove without deprecation.

Reply:
I'd be happy to merge this PR, then I'll open another PR to deprecate the feature.